### ISSUE TITLE
Very regular freezes

### ISSUE THREAD

**Issue by cromega** [brave/brave-browser#42553]
### Description

The browser hangs very regularly for what feels like always about the same amount of time, around 3 seconds. The browser locks up completely, can't interact with the UI at all.  Sometimes it happens once or twice and then it's fine for a while, other times the browser becomes completely unusable with one hang occuring right after another.

Disabling graphical acceleration resolves the problem but it's hardly a solution.

### Steps to reproduce

1. Run the browser with graphical acceleration enabled
2. Do something, have tabs open

### Actual result

1. The browser locks up for seconds regularly

### Expected result

1. The browser does not lock up for seconds regularly

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

1.73.91 Chromium: 131.0.6778.85 (Official Build) (64-bit)

OS: Tuxedo OS 4
Kernel: 6.11.0-105009-tuxedo
model: TUXEDO Pulse 14 Gen4
CPU/GPU: AMD Ryzen 7 8845HS w/ Radeon 780M Graphics

### Channel information

- [x] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by jonatino** [comment]
Yup. Brave is the ONLY chromium based browser that does this on my machine.

**Comment by tribbloid** [comment]
yep, also affect me

**Comment by djmolny** [comment]
Same symptoms here. Turning off hardware acceleration and restarting Brave eliminates the freezes.

Brave 1.81.136 / Chromium 139.0.7258.143
Ubuntu 24.04.3

**Comment by secit** [comment]
I’m encountering the same issue. Turning off hardware acceleration resolved it.

I have two machines that has been working fine for months. One started experiencing the problem, and a week later the other did as well, all without any package updates. I ran a system update hoping it would help, but it didn’t.

Brave 1.85.111 (Official Build) (64-bit)
Chromium: 143.0.7499.40
